source/modules/TestDrivenDevelopment.rst

```diff
@@ -7,20 +7,20 @@ FIXME: change the path from my personal to something generic
 Test Driven Development
 #######################
 
-"Testing" is any strategy for making sure your code behaves as expected. "Unit Testing" is a particular strategy, that:
+"Testing" is any strategy for making sure your code behaves as expected. "Unit testing" is a particular strategy, that:
 
-* is easy to run in an automated fashion.
-* utilized isolated tests for each individual
+* is easy to run in an automated fashion
+* utilizes isolated tests for each individual
```

Comment:
Should this be "each individual [something]"?

Reply:
yes indeed. should be fixed now.